CHANGELOG.md

```diff
@@ -5,6 +5,23 @@ All notable changes to this project will be documented in this file.
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
+## [5.0.1] - 2019-02-25
+
+### Features
+- Implement a new`typed-enum` rule to detect enum value that do not respect specified type [#913](https://github.com/stoplightio/spectral/pull/913)
+
+### Changed
+- API: introduced document [#876](https://github.com/stoplightio/spectral/pull/876) - note, this is **not** a breaking change
+- Introduce optional enhancers to casing function [#884](https://github.com/stoplightio/spectral/pull/884)
+
+### Fixes
+- Improved Example Object validation [#882](https://github.com/stoplightio/spectral/pull/882)
+- `oas3-operation-security-defined` rule supports optional authentication [#895](https://github.com/stoplightio/spectral/pull/895)
+- Generate more correct paths to error [#900](https://github.com/stoplightio/spectral/pull/900)
```

Comment:
The `to error` part in `Generate more correct paths to error` reads a bit weird to me. 

`When reporting an error`?

Reply:
```suggestion
- Generate more correct paths when reporting an error [#900](https://github.com/stoplightio/spectral/pull/900)
```
Sounds good!